src/test/java/org/opensearch/neuralsearch/processor/combination/RRFScoreCombinationTechniqueTests.java

```diff
@@ -63,9 +83,21 @@ public void testCombineWithNullInput() {
 
     private float RRF(List<Float> scores, List<Double> weights) {
         float sumScores = 0.0f;
-        for (float score : scores) {
-            sumScores += score;
+        float sumOfWeights = 0;
```

Comment:
we probably need to use alternative way of calculating RRF final score, this one looks like exact copy of main implementation. This can be done later thought, not in this PR